catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -13,31 +13,33 @@
  */
 package ddf.catalog.resource.download;
 
+import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.concurrent.Callable;
 import java.util.concurrent.atomic.AtomicLong;
 
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.google.common.io.CountingOutputStream;
 
 /**
- * ReliableResourceCallable is responsible for reading product data from its @InputStream and then writing that data
- * to a @FileBackedOutputStream (that will be concurrently read by a client), and optionally caching the product to 
- * the file system. It is a @Callable that is started via a @Future by the @ReliableResourceDownloadManager class.
- *
- * The client uses the @ReliableResourceInputStream to read from the @FileBackedOutputStream. 
- *
- * This class will read bytes in chunks (whose size is specified by the caller) until it either reaches the EOF or it 
- * is interrupted (either by an @IOException or the @CachedResource).
- *
- * If the @InputStream is read until the EOF, then a -1 is returned indicating the entire stream was read successfully. 
- * Otherwise, the number of bytes read thus far is returned so that the caller can react accordingly (e.g., reopen the 
- * product @InputStream and skip forward that many bytes already read).
- *
+ * ReliableResourceCallable is responsible for reading product data from its {@link InputStream} and then writing that data
+ * to a {@link CountingOutputStream} (that will be concurrently read by a client), and optionally caching the product to
+ * the file system. It is a {@link Callable} that is started via a @Future by the {@link ReliableResourceDownloadManager} class.
```

Comment:
Note that `@Future` will not show up on JAVADOCS. You'll need to do `{@link Future}` instead.